docs/kafka-cli/topics/client.md

```diff
@@ -0,0 +1,55 @@
+Use `openssl` to package up the client private key and certificate into PKCS12 format. You'll be prompted to create a password here. Hold on to this, as you'll need it in the next step and in configuration later.
+
+```shell-session
+$ openssl pkcs12 -export -in {{ client_cert }} -inkey {{ client_key }} -name {{ client_name }} > client.p12
+```
```

Comment:
Does kafka require pkcs12?
Because I've been able to import a certificate using only `step`, `keytool`, and `sed`

```sh
keytool -genkeypair -alias internal.smallstep.com -keyalg RSA -keysize 2048 -keystore kafka.jks
keytool -certreq -alias internal.smallstep.com -file internal.csr -keystore kafka.jks -ext san=dns:internal.smallstep.com
sed -i -e 's/NEW //g' internal.csr
step ca sign internal.csr internal.crt
keytool -importcert -keystore kafka.jks -alias internal.smallstep.com -file internal.crt```

Reply:
The organization and all the stuff is not coming from step-certificates, but you don't really need openssl.